go-controller/pkg/ovn/pods.go

```diff
@@ -26,27 +26,39 @@ import (
 )
 
 func (oc *Controller) syncPods(pods []interface{}) {
+	oc.syncWithRetry("syncPods", func() error { return oc.syncPodsRetriable(pods) })
```

Comment:
good opportunity to add docstring?

Reply:
heh... not really 😜 I already added it to syncWithRetry.

Reply:
It's up to you.  "syncPods will call syncWithRetry to sync pods in case of failure"?

Reply:
@dougsland   "syncPods will call "syncWithRetry" once.
 "syncWithRetry" will call "syncPodsRetriable" until it succeeds or times out.